enhancements/windows-containers/bring-your-own-windows-host.md

```diff
@@ -0,0 +1,171 @@
+---
+title: bring-your-own-windows-host
+authors:
+  - "@aravindhp"
+reviewers:
+  - "@@openshift/openshift-team-windows-containers"
+approvers:
+  - "@sdodson"
+creation-date: 2021-01-19
+last-updated: 2021-01-19
+status: implementable
+---
+
+# Bring Your Own Windows Host
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The intent of this enhancement is to allow customers to add their existing
+Windows instances to an OpenShift 4.7+ cluster using the Windows Machine Config
+Operator (WMCO).
+
+## Motivation
+
+The world of Windows sees customers treating their instances more as "pets"
```

Comment:
Just confirming, Is this for only OpenShift non-IPI clusters? I ask because ignition endpoint may not be accessible in case of other types. Calling this out early in the motivation will be helpful.

Reply:
Ignition end points will be available only on hostNetwork in case of IPI. In rest of the installation types ignition endpoints may be accessible from external networks too.